### PR TITLE
add some feature attributes to repr

### PR DESCRIPTION
**Pull request by pmeier**
```py
>>> features.Image(torch.zeros(1, 2, 2))
Image([[[0., 0.],
        [0., 0.]]], color_space=ColorSpace.GRAY)
>>> features.Image(torch.zeros(3, 2, 2))
Image([[[0., 0.],
        [0., 0.]],
       [[0., 0.],
        [0., 0.]],
       [[0., 0.],
        [0., 0.]]], color_space=ColorSpace.RGB)
>>> box = features.BoundingBox([0, 0, 2, 2], format=features.BoundingBoxFormat.XYXY, image_size=(2, 2))
>>> box
BoundingBox([0, 0, 2, 2], format=BoundingBoxFormat.XYXY, image_size=(2, 2))
>>> box.to_format(features.BoundingBoxFormat.CXCYWH)
BoundingBox([1, 1, 2, 2], format=BoundingBoxFormat.CXCYWH, image_size=(2, 2))
```

I intentionally did not add `Label.categories` here, since we need some way to shorten the list in case it is long. For example, that will be 1000 entries for ImageNet with each entry roughly 20-30 characters long.

### CODE REVIEW

**Review by vfdev-5 — APPROVED** [review]
lgtm, thanks @pmeier

**Review by datumbox — APPROVED** [review]
LGTM, thanks!
